types/api/v1/cluster_types.go

```diff
@@ -36,6 +36,7 @@ type Hosts struct {
 	CPU        string   `json:"cpu,omitempty"`
 	Memory     string   `json:"memory,omitempty"`
 	Count      string   `json:"count,omitempty"`
+	ImageID    string   `json:"imageId,omitempty"`
```

Comment:
Infra package is quite decoupled with the sealer main logic. Then when hacking infra package, we should not modify anything from sealer main package.

Will we take vm image into consideration? This should be discussed, too. Now I disagree with that.

Reply:
ok, thanks for your valuable comments，maybe there is a better way to solve this problem.